### ISSUE TITLE
Crash with nunjucks

### ISSUE THREAD

**Issue by KidsDontPlay** [fastify/point-of-view#176]
## 🐛 Bug Report

When I render a template with the server instance the program crashes.

```
/home/gaus/Projects/test/node_modules/point-of-view/index.js:329
      this.header('Content-Type', 'text/html; charset=' + charset).send(html)
                                                                  ^

TypeError: Cannot read property 'send' of undefined
    at /home/gaus/Projects/test/node_modules/point-of-view/index.js:329:67
    at /home/gaus/Projects/test/node_modules/nunjucks/src/environment.js:39:5
    at RawTask.call (/home/gaus/Projects/test/node_modules/asap/asap.js:40:19)
    at flush (/home/gaus/Projects/test/node_modules/asap/raw.js:50:29)
    at processTicksAndRejections (internal/process/task_queues.js:79:11)
```

## To Reproduce

Execute this piece of code.

```js

const fastifyServer = require("fastify").fastify({
    trustProxy: true,

}).register(require("point-of-view"), {
    engine: {
        nunjucks: require('nunjucks'),
    },
    root: path.join(__dirname, "templates"),
    options: { tags: { variableStart: '[[', variableEnd: ']]' } },
    viewExt: "html"
});
fastifyServer.listen(23232, () => {
    fastifyServer.view("api", { }, console.log);
})

```

## Expected behavior

No crash and a rendered page.

## Your Environment

- *node version*: 12.18.3
- *fastify version*: 3.3.0
- *point-of-view version*: 4.5.0
- *nunjucks version*: 3.3.2
- *os*: Linux

**Comment by mcollina** [comment]
True! There might be something off here... would you like to send a PR with a fix? Remember to add a unit test.

**Comment by Eomm** [comment]
The decorator on the fastify server should have some edge cases

As workaround this should work:

```
fastifyServer.listen(23232, (req, reply) => {
    reply.view("api", { }, console.log);
})
```

**Comment by Eomm** [comment]
Released in 4.5.1

Thanks for the PR #177
